.github/workflows/checks.yml

```diff
@@ -70,8 +70,8 @@ jobs:
           submodules: recursive
       - name: Set up environment
         uses: ./.github/actions/setup
-      - run: rm foundry.toml
-      - uses: crytic/slither-action@v0.4.0
+      - run: Slither
+        uses: crytic/slither-action@v0.4.0
```

Comment:
I see what's going on. Why remove the dependancies in `package.json`? Wouldn't this break the npm pacakge? We also shouldn't bundle it into this pr if we want to do that.

Reply:
yeah agreed -- im not sure i get this and the package.json change

Reply:
The package.json changes are because we decided to use only the `master` branch of the vanilla repository. For that, we don't need the npm version anymore and instead, we keep a single remapping per package (i.e. vanilla, upgrades). I think these should go in a different PR

I also don't know what's up with this change in particular